docker/circleci/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
+
+PROXY=""
+if [ "$https_proxy" ]; then
+	PROXY=" --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy"
+fi
+
+docker build -f $DIR/circleci.Dockerfile $DIR/../.. --tag metajack/circleci:latest --build-arg GIT_REV="$(git rev-parse HEAD)" --build-arg GIT_UPSTREAM="$(git merge-base HEAD origin/master)" --build-arg BUILD_DATE="$(date -u +'%Y-%m-%dT%H:%M:%SZ')" $PROXY
```

Comment:
Can the tag be handle-less?

Reply:
Yes, I forgot to change the tag to match the other ones. I added a fixup commit for that.